contracts/axone-cognitarium/Cargo.toml

```diff
@@ -35,10 +35,26 @@ thiserror.workspace = true
 
 [dev-dependencies]
 base64 = "0.22.1"
+cucumber = "0.21.0"
 cw-multi-test.workspace = true
+futures = "0.3.30"
+serde_json = "1.0.117"
+serde_yaml = "0.9.34"
 
 [features]
 # for more explicit tests, cargo test --features=backtraces
 backtraces = ["cosmwasm-std/backtraces"]
 # use library feature to disable all instantiate/execute/query exports
 library = []
+
+[[test]]
+harness = false
+name = "cucumber"
+path = "tests/e2e/main.rs"
+
+[package.metadata.scripts]
+optimize = """docker run --rm -v "$(pwd)":/code \
+  --mount type=volume,source="$(basename "$(pwd)")_cache",target=/code/target \
+  --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
+  cosmwasm/rust-optimizer:0.12.10
+"""
```

Comment:
```suggestion
```

This shouldn't be needed I think :)

Reply:
Oh yes, absolutely! 😅 Thanks for spotting it.
Fixed with 1771bb5.